firmware/ci.c

```diff
@@ -959,6 +960,24 @@ void ci_service(void)
 
 	token = get_token(&str);
 
+	if(strcmp(token, "") == 0) {
+		if(blank_command_counter == 1) {
```

Comment:
That int is initialised with 0.  Now means this code is ran if I connect to a newly booted Opsis and push enter once.